version: "3.3"

services:
  gmn-data-store-init:
    image: python:3.9
    restart: "no"
    entrypoint: /bin/bash
    command: -c "python -m pip install /gmn-data-store && gmn-data-store setup-database"
    volumes:
      - ./:/gmn-data-store
      - gmn_data_store:/root/.gmn_data_store
    networks:
      - gmn_data_platform_network
    profiles:
      - init

volumes:
  gmn_data_store:
    external: true

networks:
  gmn_data_platform_network:
    name: gmn_data_platform_network
